dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/ResultsParser.java

```diff
@@ -3,15 +3,150 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
+import java.io.*;
 
 public class ResultsParser {
 
+
+    public static final String PARALLELIZATION = "New order time:"; //name of line to get values from in Parallelization technique
```

Comment:
Instead of putting the comments for this variable after the declaration, please put it above it.
The reason for this is to somewhat enforce that each line should be no more than 100 characters.
This applies to the APFD_VALUE variable too.